### ISSUE TITLE
Homepage Home link lacks aria-current="page"

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#28]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
[Developer Portal van de Nederlandse Overheid](https://don.apps.digilab.network/)

**Probleembeschrijving:**
Voeg op de Homepage een aria-current="page" toe aan de "Home" link zodat de gebruiker weet dat ze zich momenteel op de pagina bevinden waar de link naartoe zou leiden. Het bevestigt dat ze al op de pagina zijn waar de link naar verwijst.

**Referentiecode:**
`<a class="navbar__brand" href="/"><b class="navbar__title text--truncate">Home</b></a>`

**Gerelateerde WCAG:**
WCAG Best Practice
WCAG 2.4.8 Location

**Referentieafbeelding:**
<img width="618" alt="screenreader focused on a link fails to mention the link is to the current page due to a lack of aria-current attribute" src="https://github.com/user-attachments/assets/952a48b3-bcbd-4aec-a685-eb802fa27711" />

_**Referentienummer:** A049_
